### PR TITLE
[v6.2] Edit Pass #8

### PR DESCRIPTION
**Pull request by inertial-frame**
This PR picks up where the previous Edit Pass left off.

#6943

---

Improves landing pages for the Database Section. Rest of the Database and Cloud Sections as well.

### CODE REVIEW

**Comment by inertial-frame** [comment]
Build errors above appear due to these links: https://nvd.nist.gov/800-53/Rev4/control/AC-3 which display:

Screen Shot 2021-05-21 at 8 15 45 PM

My guess is that it's some server issue. I've reported this issue to them as well.

File: docs/pages/database-access/guides.mdx [hunk 1]
```diff
@@ -3,12 +3,25 @@ title: Database Access Guides
 description: Guides for configuring Teleport Database Access with self-hosted or AWS RDS/Aurora PostgreSQL and MySQL
 ---
 
-# Database Access Guides
+# Guides
```
Comment (inertial-frame) [inline docs/pages/database-access/guides.mdx:6]: Note: remove this and made tocless

File: docs/pages/cloud/architecture.mdx [hunk 2]
```diff
@@ -1,40 +1,37 @@
...
 
-We have designed the Teleport Cloud environment to be secure; however we are still in the process of
-scrutinizing and executing on our security roadmap and working with independent security
-auditors to identify any gaps. Only once this is complete, the team will evaluate
-whether the Teleport Cloud is ready for strict compliance use-cases.
+We have designed the Teleport Cloud environment to be secure; and, security is an ongoing process of improvement.
+
+Our team routinely scrutinizes, reviews, updates, improves upon, and executes our security roadmap by working with independent security auditors to identify any gaps.
+
+Teleport Cloud only gets the green light for use in strict compliance use-cases only when the team has full confidence that all security gaps or vulnerabilities have been identified and resolved.
```
Comment (benarent) [inline docs/pages/cloud/architecture.mdx:12]: Should we change the naming of Teleport Cloud etc, to better align with company messaging?

File: docs/pages/cloud/architecture.mdx [hunk 3]
```diff
@@ -1,40 +1,37 @@
...
 
 ## Compliance
 
-We are going through SOC2 audit that should wrap up by June 2021.
+A SOC2 audit is presently underway and should be concluded by June 2021.
```
Comment (benarent) [inline docs/pages/cloud/architecture.mdx:16]: Can we just update it with the SOC2 audit? @TravisGary

File: docs/pages/cloud/faq.mdx [hunk 4]
```diff
@@ -67,14 +69,14 @@ cluster on AWS.
...
+We will add an option to trigger upgrades manually and automatically within a configurable maintenance window.
 
 ### Does your SOCII report include Teleport Cloud?
 
-Our current SOCII Type2 report covers our organization, Teleport and Gravity OSS and Enterprise versions.
-We are pursuing SOC2 for cloud and expect to receive our report at the end of Q2 2021
-after our 6 month audit period is completed.
+Our current SOCII Type2 report covers our organization, Teleport and Gravity Open Source, and Enterprise Enterprise.
+
+We are pursuing SOC2 for Cloud and expect to receive our report at the end of Q2 2021 after our 6 month audit period is completed.
```
Comment (benarent) [inline docs/pages/cloud/faq.mdx:79]: Update with latest SOC2 info here.

**Review by benarent — APPROVED** [review]
These edit commits are bit of a beast to review but it looks good.

File: docs/pages/cloud/faq.mdx (hunk 4 above)
Comment (inertial-frame) [reply docs/pages/cloud/faq.mdx:79]: Yeah, let's - at least for 7.0: https://github.com/gravitational/teleport/issues/7186 - should we update this for 6.2 as well?

Comment (inertial-frame) [reply docs/pages/cloud/faq.mdx:79]: Hmmm ... We can also remove this section and link to a new Architecture/Security article section. (Or refactor it into an include so it lives in both places.) 

You have a preference about the best way here? Will we have video or blog content to support that? Seems like a cool thing to detail a bit - plus there seems to be some confusion about Enterprise and Open Source offerings presently. Maybe something that tackles all of those?

File: docs/pages/cloud/architecture.mdx (hunk 3 above)
Comment (klizhentas) [reply docs/pages/cloud/architecture.mdx:16]: SOC2 audit has been completed for the cloud officially. We can state it now.

File: docs/pages/cloud/faq.mdx [hunk 5]
```diff
@@ -50,14 +52,14 @@ After [connecting](#how-can-i-access-the-tctl-admin-tool) `tctl` to Teleport Clo
 [dynamic tokens](../admin-guide.mdx#short-lived-dynamic-tokens):
 
 ```bash
-$ tctl nodes add --ttl=5m --roles=node,proxy --token=$(uuid)
+tctl nodes add --ttl=5m --roles=node,proxy --token=$(uuid)
 ```
 
 ### When will a security audit be available?
 
 The Teleport Cloud security audit starts in January.
```
Comment (klizhentas) [inline docs/pages/cloud/faq.mdx:60]: The audit has been completed on April 12th:

https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/

**Review by klizhentas — APPROVED** [review]
Let's update cloud SOC2 and Audit status, otherwise LGTM

File: docs/pages/cloud/architecture.mdx (hunk 3 above)
Comment (inertial-frame) [reply docs/pages/cloud/architecture.mdx:16]: @benarent , @klizhentas - I've synced with @TravisGary - a new blog post has been created (will be published) that we can link to in this section along with a brief snippet.

I believe there's an Issue up to more comprehensively address this in 7.0 - https://github.com/gravitational/teleport/issues/7186 - we'll probably want to update the FAQ, move this section into an include, reuse it in a new section within the Security article. 

I'll hold off on merging this until that's done if that's OK. Should be good here shortly I'd think.

File: docs/pages/cloud/faq.mdx (hunk 5 above)
Comment (inertial-frame) [reply docs/pages/cloud/faq.mdx:60]: Ticket tracking this: https://github.com/gravitational/web/issues/1893

File: docs/pages/cloud/architecture.mdx (hunk 2 above)
Comment (inertial-frame) [reply docs/pages/cloud/architecture.mdx]: Done.

**Comment by inertial-frame** [comment]
@benarent , @klizhentas - It's my day off but I wanted to get these changes in!

I went through and updated all Documentation references to `Teleport Cloud`. Redirects and text updated to `Teleport Pro`. I'll have two small PR's that'll address some of the PR Templating and Code comments/messaging.

Also, I've made a small include for SOC 2 info. We can add the new blog for https://github.com/gravitational/web/issues/1893 easily. https://github.com/gravitational/teleport/issues/7186 is addressed by this.

**Review by benarent — APPROVED** [review]
👍

**Pull request merged**